File: src/test/java/application/browser/StatusHelper.java
package application.browser;

import static org.openqa.selenium.support.ui.ExpectedConditions.textToBe;

import org.openqa.selenium.By;

public class StatusHelper {
  private MatagBrowser matagBrowser;

  StatusHelper(MatagBrowser matagBrowser) {
    this.matagBrowser = matagBrowser;
  }

  public void hasMessage(String message) {
    matagBrowser.wait(textToBe(By.id("status-message"), message));
  }
}

File: src/test/kotlin/application/browser/StatusHelper.kt
package application.browser

import org.openqa.selenium.By
import org.openqa.selenium.support.ui.ExpectedConditions

class StatusHelper internal constructor(private val matagBrowser: MatagBrowser) {
    fun hasMessage(message: String) {
        matagBrowser.wait(ExpectedConditions.textToBe(By.id("status-message"), message))
    }
}
